packages/platform/platform-fastify/src/components/PlatformFastify.ts

```
@@ -0,0 +1,337 @@
+        request: request as any,
+        response: reply as any
+      });
+
+      ($ctx.request.getReq() as any).$ctx = $ctx;
+
+      await $ctx.start();
+    });
+
+    rawApp.addHook("onResponse", async (request, reply) => {
+      const {$ctx} = request;
+      delete ($ctx.request.getReq() as any).$ctx;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)